### DISCUSSION TITLE
Excluding birds

### DISCUSSION THREAD

**Discussion by PeterIP4** [mcguirepr89/BirdNET-Pi#534] [Ideas]
First of all thank you for an excellent product.
I am getting quite a few spurious birds logged such as Atlantic Puffin.
When I go to Tools -> Excluded species I am presented with a huge list of birds to chose. This list is in scientific name order and not the way in which the name is shown elsewhere Fratercula Arctica_Atlantic Puffin.
It would be nice to be able to search by common name rather than look up the scientific name and then scroll down.
BTW page don only moves me a few lines up and don the list.

**Comment by mcguirepr89** [comment]
Hi, @PeterIP4  -- thank you for the observations -- I recommend using your browser's "Find" tool -- pressing "Ctrl+F" will allow you to search the list by the species' common name.
